.gitignore

```diff
@@ -1,2 +1,5 @@
 # Ignore build or dist files
 /elm-stuff
+
+# Ignore IntelliJ config files
```

Comment:
I think you can remove this from the patch.

Reply:
Fixed